next.config.js

```diff
@@ -205,7 +205,7 @@ const sentryOptions = {
   hideSourceMaps: false,
 
   sourcemaps: {
-    disable: process.env.UPLOAD_SENTRY_SOURCEMAPS !== "true",
```

Comment:
I think this is fine, IIRC it will log a warning saying what you need to do (e.g. set the token) but should not break the build.

Reply:
Ah right, for people who don't have the token, having it on by default will log the error, that makes sense. But agreed that that's no biggie. If we really want to disable the warning, we can also just trigger `disable` based on whether the token is available - but I think it'd be good to have the warning in our CI logs if something goes wrong with the env var.